include/zephyr/net/wifi.h

```diff
@@ -655,6 +676,14 @@ enum wifi_ap_config_param {
 	WIFI_AP_CONFIG_PARAM_MAX_INACTIVITY = BIT(0),
 	/** Used for AP mode configuration parameter max_num_sta */
 	WIFI_AP_CONFIG_PARAM_MAX_NUM_STA = BIT(1),
+	/** Used for AP mode configuration parameter bandwidth */
+	WIFI_AP_CONFIG_PARAM_BANDWIDTH = BIT(2),
+#if defined(CONFIG_WIFI_NM_WPA_SUPPLICANT)
```

Comment:
any reason for this guard? These could be applicable to offload too?

Reply:
Yes, no need.

Reply:
> Currently this parameter is related to hostap. Should we consider wrapping it with `CONFIG_WIFI_NM_HOSTAPD_AP` macro?

@krish2718, what do you think?

Reply:
Sure, WPA supplicant based AP doesn't provide these anyway.